index.html

```diff
@@ -1,90 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Deleting this file doesn't reroute the website since everything that's actually live is in the `/new/` directory.